.github/dependabot.yml

```diff
@@ -57,7 +57,7 @@ updates:
 
   - package-ecosystem: "gitsubmodule"
     directory: "/"
-    labels: ["deps", "not ready"]
+    labels: ["deps"]
```

Comment:
Let's keep it. Otherwise, dependabot PR rebases consume all CI runners for the whole organization, making FerretDB development much slower.